.gitignore

```diff
@@ -28,4 +28,3 @@ _testmain.go
 
 /build/
 coverage.out
-/vendor/
```

Comment:
I usually keep this so `go mod vendor` works

Reply:
I don't usually use that so I missed that use case. Have reverted this and the .dockerignore, thanks.

Reply:
I don't usually use that so I missed that use case. Have reverted this and the .dockerignore, thanks.